src/utils/customMessages.js

```diff
@@ -51,7 +49,11 @@ const messages = {
   usernameExistOrEmpty: 'The username you entered already exists or is empty, if you want to continue update your profile, change the new username or unselect it to remain with the old one',
   passwordChangeSuccess: 'Your password has been changed, login with new password',
   changeEmailNotAllowed: 'You cannot change your email, unselect it if you want to continue updating your profile, otherwise you will not be able to continue',
-  tokenInvalid: 'Authentication failed. Token is invalid'
+  tokenInvalid: 'Authentication failed. Token is invalid',
+  oneWayTripRequestFailed: 'trip request request failed',
+  multiCitiesTripRequestCreated: 'Multiple cities trip request successfuly submitted and waiting for approval',
+  oneWayTripRequestCreated: 'trip request successfuly submitted and waiting for approval',
+  dataLengthErr: 'request should not be more than 2'
```

Comment:
is a user only allowed to create 2 trips?

Reply:
sure, maximum 2